native_client/java/libmozillavoicestt/src/main/java/org/mozilla/voice/stt/MozillaVoiceSttStreamingState.java

```diff
@@ -0,0 +1,13 @@
+package org.mozilla.voice.stt;
```

Comment:
Why is this file added? It looks like to be SWIG-generated

Reply:
https://github.com/mozilla/DeepSpeech/blob/master/native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech/DeepSpeechStreamingState.java

Reply:
maybe it was my mistake :)